src/Http/Http/test/HttpContextAccessorTests.cs

```diff
@@ -27,7 +21,7 @@ public async Task HttpContextAccessor_GettingHttpContextReturnsHttpContext()
 
             await Task.Delay(100);
 
-            Assert.Same(context, accessor.HttpContext);
+            Assert.True(accessor.HttpContext.Equals(context));
```

Comment:
These will give you a very poor message if they ever start failing. "Expected true, was false". I'd add a specific message unless you can find an assert that actually uses the Equals method.

Reply:
Done